components/formats-api/src/loci/formats/IFormatWriter.java

```diff
@@ -201,4 +201,36 @@ void savePlane(int no, Object plane, Region tile)
    */
   void setWriteSequentially(boolean sequential);
 
+  /**
+   * Retrieves the current tile width
+   * Defaults to full image width if not supported
+   * @return The current tile width being used
+   * @throws FormatException Image metadata including Pixels Size X must be set prior to calling getTileSizeX()
+   */
+  int getTileSizeX() throws FormatException;
+
+  /**
+   * Will attempt to set the tile width to the desired value and return the actual value which will be used
+   * @param tileSize The tile width you wish to use
+   * @return The tile width which will actually be used, this may differ from the value attempted to set
```

Comment:
"attempted to set" could be "requested", and maybe also mention the reasons why the returned value might differ e.g. if the writer doesn't support it.